packages/react-magma-dom/src/components/Tabs/utils.ts

```diff
@@ -13,6 +13,8 @@ export function useTabsMeta(theme, orientation, backgroundColor, isInverse) {
   const scrollStart = vertical ? 'scrollTop' : 'scrollLeft';
   const clientSize = vertical ? 'clientHeight' : 'clientWidth';
 
+  const approxTabSize = vertical ? 80 : 120;
```

Comment:
should these be in the theme?

Reply:
Oh!  Good call.  I hated hard-coding them, but thought it would be too much to calculate them dynamically.  Putting them in the theme might be a good compromise.